Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -612,6 +612,7 @@ class BrowserViewController: UIViewController {
         [header, footer, readerModeBar].forEach { view in
                 view?.transform = CGAffineTransform.identity
         }
+        self.statusBarOverlay.isHidden = false
```

Comment:
nit: `self`. I actually have to expand the github view when I see this to confirm the code is not in a closure.